rules/linux/credential_access_potential_linux_ssh_bruteforce_external.toml

```diff
@@ -0,0 +1,89 @@
+[metadata]
+creation_date = "2022/09/14"
+integration = ["system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/03/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies multiple external consecutive login failures targeting a user account from the same source address within 
+a short time interval. Adversaries will often brute force login attempts across multiple users with a common or known
+password, in an attempt to gain access to these accounts.
+"""
+from = "now-9m"
+index = ["logs-system.auth-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential External Linux SSH Brute Force Detected"
+note = """## Triage and analysis
+
+### Investigating Potential External Linux SSH Brute Force Detected
+
+The rule identifies consecutive SSH login failures targeting a user account from the same source IP address to the same target host indicating brute force login attempts.
+
+This rule will generate a lot of noise for systems with a front-facing SSH service. 
+Adversaries scan the whole internet for remotely accessible SSH services, and try to brute force them in order to gain unauthorized access. 
+In case this rule generates too much noise, and external brute forcing is of not much interest, consider turning this rule off and enabling "Potential Internal Linux SSH Brute Force Detected" to detect internal brute force attempts.
```

Comment:
```suggestion
The rule identifies consecutive SSH login failures targeting a user account from the same source IP address to the same target host indicating brute force login attempts.

This rule will generate a lot of noise for systems with a front-facing SSH service, as adversaries scan the internet for remotely accessible SSH services and try to brute force them to gain unauthorized access. 

In case this rule generates too much noise and external brute forcing is of not much interest, consider turning this rule off and enabling "Potential Internal Linux SSH Brute Force Detected" to detect internal brute force attempts.
```